### ISSUE TITLE
Create documentation about how to create a backend service 

### ISSUE THREAD

**Issue by offtherailz** [geosolutions-it/MapStore2#9104]
## Description

*Documentation section involved*
- [ ] User Guide
- [ ] Developer Guide

## Other useful information

In a project I created a "module" system that could be ported to simplify the development of new features on back-end, for both main product and dependent products.

Basically this is a module. 

https://github.com/geosolutions-it/npa-cgg/tree/main/front-end/satExplorer/backend/modules

basically it is simply implemented ad an dedicated component-scan for Spring MVC controllers, so the only thing to do is to add some  classes to the package to create an entry point ready to go.

https://github.com/geosolutions-it/npa-cgg/blob/main/front-end/satExplorer/web/src/main/webapp/WEB-INF/modules-servlet.xml
